http-server/src/access_control/mod.rs

```diff
@@ -41,6 +41,8 @@ use jsonrpsee_utils::http_helpers;
 pub struct AccessControl {
 	allow_hosts: AllowHosts,
 	cors_allow_origin: Option<Vec<AccessControlAllowOrigin>>,
+	// TODO: (dp) We never set this header in the response headers.
+	#[allow(dead_code)]
 	cors_max_age: Option<u32>,
```

Comment:
☝️

Reply:
I think we can drop it? The defaults in browsers seem sensible, and I don't suppose there really is a need to set them higher (it also appears you can't set them higher than whatever arbitrary cap a browser decides) or disable them. For most cases I assume ctrl+shift+R clears it anyway so it should be irrelevant for devs.

Reply:
from what I understand `a preflight request must be sent as a HTTP Option request`?

jsonrpc crate seems to set it for all requests https://github.com/paritytech/jsonrpc/blob/master/http/src/handler.rs#L558-L563?!

Reply:
Yes, if cached the browser will not send the preflight and either go straight to the HTTP request (if origin is allowed in cache) or block the request (if origin is not allowed in cache).

Reply:
So to do this properly we'd need to keep some state so we know if a connecting host has to do the preflight or not?

Reply:
I'm going to revert this change in this PR and remove it in #466 instead.